chrome-extension/test/configure.js

```diff
@@ -9,21 +10,21 @@ ExtensionDriver.prototype.navigate = function(url) {
 
 var id = 'bpngoepojmffegnjicpfjcakgajpmenk';
 
-before(function() {
+before(function(done) {
   this.timeout(20000);
   this.environment = 'chrome';
 
-  chrome.setDefaultService(
-    new chrome.ServiceBuilder(chromeDriver.path).build()
-  );
+  var test = this;
 
-  var options = new chrome.Options();
-  options.addExtensions(path.resolve('chrome-extension/dist/tipsy.crx'));
-
-  var driver = chrome.createDriver(options);
+  var service = new chrome.ServiceBuilder(chromeDriver.path).build();
+  chrome.setDefaultService(service);
 
-  driver.manage().timeouts().implicitlyWait(10000);
-  this.extensionDriver = new ExtensionDriver(driver, id);
+  var options = new chrome.Options();
+  options.addExtensions(path.resolve('chrome-extension/dist/tipsy.crx'), function() {
+    test.driver = new chrome.Driver(options, service);
 
-  return this.extensionDriver.navigate('html/index.html');
+    //driver.manage().timeouts().implicitlyWait(1000);
+    test.extensionDriver = new ExtensionDriver(test.driver, id);
+    test.extensionDriver.navigate('html/index.html').then(done);
+  });
 });
```

Comment:
David,

This looks reasonable (but I'm not familiar with this interface). If it passes, it should be good.